app/models/concerns/orchestration/compute.rb

```diff
@@ -158,6 +152,20 @@ def setComputeIP
 
   def delComputeIP; end
 
+  def setComputeIPAM
+    set_ip_address
+
+    unless required_ip_addresses_set?(false)
+      failure _('Failed to set IPs via IPAM for %s') % name
```

Comment:
It'd be useful to include more debug here to indicate which IP(s) is expected + missing, but I'm not certain on the best method. Perhaps by adding errors to the model in the `required_ip_addresses_set?` method and ensuring they're logged?